### ISSUE TITLE
Undeclared variable does not trigger error.

### ISSUE THREAD

**Issue by jeffmcmahan** [AtomLinter/linter-jshint#168]
```js
;(function () {

  'use strict'

  implicitGlobal = 4 
  // No warning from linter-jshint, but jshint.com throws undefined variable warning.

}())
```

Is there a default that I need to override? I don't think this behavior was standard a short while ago. Seems like this is caused by -W117 being set somewhere, but I cannot seem to fix it with .jshintrc, much less as a default option.

**Comment by piercers** [comment]
I think you're looking for [`undef`](http://jshint.com/docs/options/#undef).

**Comment by Arcanemagus** [comment]
Closing based on the comment above.
